### ISSUE TITLE
Adding 7th App to config.json and pressing down arrow for next page causes crash

### ISSUE THREAD

**Issue by xtacocorex** [NextThingCo/PocketCHIP-pocket-home#58]
In the IRC chat, I had asked what happens if you add a 7th app to config.json.  nbastin agreed to test.  Upon restarting pocket-home, a down arrow was presented for the next page.

When nbastin clicked the down arrow, pocket-home crashed.  There appear to be no logs from the crash.  I currently do not have my PocketCHIP, so I cannot recreate to look for any logs.

We do know that hacking config.json at this moment in time is like playing with fire.  We also figured you all know about this bug. :)

**Comment by ahk** [comment]
I didn't actually. I hadn't tested it in a while and apparently in the shipped version it's broken as you say. It's been fixed for several weeks in one of our upcoming feature branches (which adds dynamic rescaling for arbitrary aspect displays, aka you can run pocket-home as an alternative desktop UI). I have also realized I need to start watching github issue more closely.

Thanks for the report, and sorry about this bug. config.json was meant to be edited by people who find it.

**Comment by xtacocorex** [comment]
Ah, cool features upcoming too!  It should be pretty easy to clone this repo and build/replace the current pocket-home on ones PocketCHIP right?

**Comment by ahk** [comment]
Yes though our README right now is almost insulting. Will flesh it out soon now that people are trying to build it. We're juggling a lot of things and the open sourcing effort has been pretty no-frills for some of these repositories.

**Comment by voltagex** [comment]
@ahk, is the fix available in the ahk/scalable-layout branch?

**Comment by marukka** [comment]
So what is required to build it? It looks like it is referencing dependencies which aren't included:

```
chip@chip:~/PocketCHIP-pocket-home-master$ make all
cd Builds/LinuxMakefile && \
sed -i '2946s/(styleFlags & windowIsSemiTransparent) ? 32 : 24/32/' ../../deps/JUCE/modules/juce_gui_basics/native/juce_linux_Windowing.cpp && \
make
sed: can't read ../../deps/JUCE/modules/juce_gui_basics/native/juce_linux_Windowing.cpp: No such file or directory
Makefile:16: recipe for target 'all' failed
make: *** [all] Error 2
```
